src/Symfony/Component/Form/ReversedTransformer.php

```diff
@@ -18,6 +18,8 @@
  * reverseTransform() method is called and vice versa.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @implements DataTransformerInterface<mixed, mixed>
```

Comment:
This could actually be more precise, by making it a generic class implementing `DataTransformerInterface<T, R>` and expecting a `DataTransformerInterface<R, T>` in the constructor.